### DISCUSSION TITLE
Favicon Configurable

### DISCUSSION THREAD

**Discussion by varioous** [firefly-iii/firefly-iii#4319] [Firefly III]
Hello,

first thanks for your great work and application.

It would be great to make the Favicon configurable under Settings. I can change them via public Folder, but after each update i have to replace them again.

Thanks and kind regards,
Harald Holzmann

**Comment by JC5** [comment]
Why?

**Comment by varioous** [comment]
Because it looks more professional und branded with own Logo/Favicon :) I would say it is nice to have.

**Comment by JC5** [comment]
A new logo is in the works. But it won't be configurable.
